### DISCUSSION TITLE
PDF preview handler can't show files that are missing specific permissions?

### DISCUSSION THREAD

**Discussion by highend** [sumatrapdfreader/sumatrapdf#3625] [Q&A]
Hi,

neither the stable, nor the current pre-release pdf preview handler can render .pdf files that are missing these 4 file permissions:
```
Create files / write data
Create folders / append data
Write attributes
Write extended attributes
```
I have no idea why a `preview` handler would need these^^

Does anyone know if this can be easily "fixed" in the source code?

Thanks!

**Comment by highend** [comment]
This issue is about file permissions (for a pdf file) that are required for the pdf preview handler to be able to render it. I've changed the wording from "security" to "file" in the original post to make this more clear.

**Comment by GitHubRulesOK** [comment]


**Reply by highend** [reply]
File permissions = The permissions that user / groups have on folders / files inside a filesystem. In this case it's NTFS.

E.g.:
![permissions](https://github.com/sumatrapdfreader/sumatrapdf/assets/687706/ec010505-4ffb-450d-9349-82d420538b40)

**Reply by GitHubRulesOK** [reply]
Sorry My bad so its a File System issue  well one task a previewer needs is to convert PDF into a Screen image see if this file previews as it needs no decryption
https://docs.google.com/gview?url=https://github.com/GitHubRulesOK/MyNotes/raw/master/colours.pdf&amp;embedded=true

however if I set file permisions for "USERS" to none I can still use Preview as its my personal rights, but you are in an area outside of SumatraPDF control.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/55c4abf5-8807-41b1-86e0-0f893011e547)
